geode-for-redis/src/integrationTest/java/org/apache/geode/redis/CommandPipeliningIntegrationTest.java

```diff
@@ -15,12 +15,15 @@
 
 package org.apache.geode.redis;
 
+import static org.apache.geode.distributed.ConfigurationProperties.GEODE_FOR_REDIS_REPLICA_COUNT;
+
 import org.junit.ClassRule;
 
 public class CommandPipeliningIntegrationTest extends AbstractCommandPipeliningIntegrationTest {
 
   @ClassRule
-  public static GeodeRedisServerRule server = new GeodeRedisServerRule();
+  public static GeodeRedisServerRule server =
+      new GeodeRedisServerRule().withProperty(GEODE_FOR_REDIS_REPLICA_COUNT, "0");
```

Comment:
Could you rather set this as the default in `GeodeRedisServerRule`? (It would also reduce the size of this PR).

Reply:
done